README.md

```diff
@@ -12,19 +12,20 @@ sudo python3 usbkill.py
 
 ### Why?
 
-In case the police or other thugs come busting in (or steal your laptop from you when you are at a public library as happened to Ross). You can use this as an anti-forensic measure. 
+There are 2 reasons (maybe more?) to use this tool:
 
-The police commonly uses a « [mouse jiggler](http://www.amazon.com/Cru-dataport-Jiggler-Automatic-keyboard-Activity/dp/B00MTZY7Y4/ref=pd_bxgy_pc_text_y/190-3944818-7671348) » to keep the screensaver and sleep mode from activating.
+- In case the police or other thugs come busting in (or steal your laptop from you when you are at a public library as happened to Ross). The police commonly uses a « [mouse jiggler](http://www.amazon.com/Cru-dataport-Jiggler-Automatic-keyboard-Activity/dp/B00MTZY7Y4/ref=pd_bxgy_pc_text_y/190-3944818-7671348) » to keep the screensaver and sleep mode from activating.
+- You don't want that someone you don't know to install backdoors and malwares on your computer / retrieve documents from your computer via USB.
```

Comment:
I think:

"You don't want that someone to install backdoors or malware on your computer or to retrieve documents from your computer via USB."

Sounds better.